index.html

```diff
@@ -190,16 +195,16 @@ <h2>Multi Post Stories</h2>
                 <li>2015</li>
               </ul>
               <p>
-                A daily selection of privately personalized; no accounts or
-                sign-ups required.
+                A smart assistant to make driving more safe, efficient, and fun
+                by unlocking your most expensive computer: your car.
               </p>
               <ul class="languages">
                 <li>html</li>
                 <li>css</li>
                 <li>javascript</li>
```

Comment:
- [ ] 4. Kindly ensure that the dots in between the items are fixed as it is in the design, this will ensure a pixel-perfect design 😄.